README.md

```
@@ -858,3 +858,94 @@ MIT
+| Category | Examples |
+|----------|----------|
+| **Secrets** | Hardcoded Anthropic API keys, tokens |
+| **Permissions** | Overly permissive allow rules like `Bash(*)` |
+| **Hooks** | Hook injection vectors, unsafe hooks |
+| **MCP Servers** | MCP server risks, malicious MCP configs |
+| **Agents** | Agent prompt injection vectors |
+
+### How to get started?
+
+**Quick Start (no install):**
+```bash
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)